CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/docs/expressions.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>The Constellation Expressions Framework</title>
+        <link rel="stylesheet" href="nbdocs://au.gov.asd.tac.constellation.preferences/au/gov/asd/tac/constellation/preferences/constellation.css" type="text/css">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    </head>
+    <body>
+        <h1>The Constellation Expressions Framework</h1>
+        <p>
+            The expressions framework is a structure which can be used to query or assign the Vertex and Transaction attributes within a graph.
+            The framework will work on all attributes which <strong>do not</strong> include a restricted character within it's AttributeName. An expression can 
+            return <strong>0..*</strong> (none or many) matches.
+        </p>
+        
+        <h2>Query Language</h2>
+        <p>
+            The query language is designed to be robust with respect to features such as nesting and type casting. To allow this, plain text is
+            used to gain the flexibility necessary. Users will be responsible for forming queries which adhere to the below rules.
+        </p>
+        
+        <h2>Forming a Query Expression</h2>
+        <p> A valid query for a vertex or transaction takes the form of <strong>[AttributeName] [Operator] [value]</strong>. </p>
+        <p>The <strong>AttributeName</strong> used will not include any restricted characters, and will be case sensitive.</p>
+        <p>The <strong>AttributeName</strong> will be represented as plain, unquoted text.</p>
+        <p>The <strong>Operator</strong> must be separated by atleast one whitespace character.</p>
+        <p>The <strong>Operator</strong> must be valid and from the list specified below.</p>
+        <p>The <strong>Value</strong> must be quoted using single('value') or double quotes("value").</p>
```

Comment:
- replace all the strong tags here with double-quotes
- for all bar the first paragraph, I reckon convert the paragraphs to an unordered list